agent/config/config.go

```diff
@@ -83,6 +83,10 @@ const (
 	// has been pulled before it can be deleted.
 	DefaultImageDeletionAge = 1 * time.Hour
 
+	// DefaultNonECSImageDeletionAge specifies the default value for minimum amount of elapsed time after an image
+	// has been created before it can be deleted
+	DefaultNonECSImageDeletionAge = 1 * time.Hour
```

Comment:
do we need a new constant for this?

Reply:
I will combine this with the constant above together.

Reply:
imo, it makes sense to have them separate, since they have different semantics (pulledAt/ createdAt)

Reply:
will make this separate since the nonECS image removal track the created time.